File: async-scala-client/src/main/scala/io/swagger/client/model/TransactionForGoodsOut.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class TransactionForGoodsOut (
  header: Option[HeaderForGoods],
lines: Option[List[LineForGoods]],
calculatedTaxSummary: Option[CalculatedTaxSummaryForGoods],
processingInfo: Option[ProcessingInfo]
)

File: javascript-closure-angular-client/API/Client/TransactionForGoodsOut.js
goog.provide('API.Client.TransactionForGoodsOut');

/**
 * @record
 */
API.Client.TransactionForGoodsOut = function() {}

/**
 * @type {!API.Client.HeaderForGoods}
 * @export
 */
API.Client.TransactionForGoodsOut.prototype.header;

/**
 * @type {!Array<!API.Client.LineForGoods>}
 * @export
 */
API.Client.TransactionForGoodsOut.prototype.lines;

/**
 * @type {!API.Client.CalculatedTaxSummaryForGoods}
 * @export
 */
API.Client.TransactionForGoodsOut.prototype.calculatedTaxSummary;

/**
 * @type {!API.Client.ProcessingInfo}
 * @export
 */
API.Client.TransactionForGoodsOut.prototype.processingInfo;

